src/components/Banner.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import styles from "../styles/Banner.module.css";
+
+const Banner = ({ bannerInfo, index, activeBanner }) => {
```

Comment:
Adicional, en el nombre de los componentes debe tener la palabra _component_ y ya sea que decidas manejarlo con class components o con functional components, debes apegarte a la definición correcta
[https://reactjs.org/docs/components-and-props.html](https://reactjs.org/docs/components-and-props.html)